src/index.ts

```diff
@@ -0,0 +1,257 @@
+/*!
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+import * as extend from 'extend';
+import * as util from 'util';
+import {Writable} from 'stream';
+const logging = require('@google-cloud/logging');
```

Comment:
`@google-cloud/logging` type has not been defined yet. This can be changed once logging was migrated.


 TS7016: Could not find a declaration file for module '@google-cloud/logging'. 
'.../nodejs-logging-bunyan/node_modules/@google-cloud/logging/src/index.js' implicitly has an 'any' type.
  Try `npm install @types/@google-cloud/logging` if it exists or add a new declaration (.d.ts) file containing `declare module '@google-cloud/logging';`

22 import logging = require('@google-cloud/logging');